circle.yml

```diff
@@ -7,9 +7,13 @@ dependencies:
   override:
     - curl -s -O https://raw.githubusercontent.com/atom/ci/master/build-package.sh
     - chmod u+x build-package.sh
+    - composer --version
     - composer config -g github-oauth.github.com cc4a091c096e7d3cfe053c3f669fb840be60ab98
     - composer global require "squizlabs/php_codesniffer=*"
 
 machine:
   php:
     version: 7.0.4
+  environment:
+    COMPOSER_DISABLE_XDEBUG_WARN: 1
+    PATH: "$HOME/.config/composer/vendor/bin:$PATH"
```

Comment:
Sorry I took so long to get back to this, I've updated the PR to use that and I'm just waiting on CI to finish before merging. This should fix things up for external PR's again.